samples/bluetooth/mesh_provisioner/prj.conf

```
@@ -1,6 +1,9 @@
 #CONFIG_INIT_STACKS=y
-CONFIG_MAIN_STACK_SIZE=2048
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_MAIN_STACK_SIZE=4392
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4740
+CONFIG_BT_MESH_SETTINGS_WORKQ_STACK_SIZE=1664
+CONFIG_BT_MESH_ADV_STACK_SIZE=3960
+CONFIG_BT_RX_STACK_SIZE=3288
```

```suggestion
CONFIG_MAIN_STACK_SIZE=4400
CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4800
CONFIG_BT_MESH_SETTINGS_WORKQ_STACK_SIZE=1700
CONFIG_BT_MESH_ADV_STACK_SIZE=4000
CONFIG_BT_RX_STACK_SIZE=3300
```

I just rounded it up to next hundred. If not, we will definitely get questions on how did we calculate exact number, as those numbers to veryy specific down to unit place. 

Now, we can say, we experiemented, added some 10% (or something, I don't know how much margin did you add), and rounded it up to next hundred.

What do you think? Does anyone else has any other opinion?

---

Sorry, I forgot to mention - the values are set with 50% margin, based off the thread analysis. With a lot for testing it seems that such margin was needed due to spikes in usage during provisioning. Rounding up is fine with me.

---

Did you want me to add a comment in the file on how the values were set?